lib/index.js

```diff
@@ -68,4 +76,14 @@ function hte (client, service, method, datas) {
   }
 }
 
+hte.Request = s.Object({
+  headers: s.Array([ s.String() ]),
+  url: s.String(),
+  method: s.String(),
+  params: s.Object({ strict: false }),
+  query: s.Object({ strict: false })
+}, {
+  strict: false
```

Comment:
If you don't have `strict: false` then if the request has anything that doesn't have a schema specific, it would be invalid. This disables that restriction